girder/models/upload.py

```diff
@@ -355,7 +355,7 @@ def createUpload(self, user, name, parentType, parent, size, mimeType=None,
             upload['reference'] = reference
 
         if parentType and parent:
-            upload['parentType'] = parentType.lower()
```

Comment:
If this is a plugin-defined model, then `parentType` refers to a list. I look at the various places that call `createUpload`, and they all sanitize the model type (typically by only allowing enumerated values) before passing it in.